### PR TITLE
UPI - Show disclaimer when Continue is pressed without a selection

### PR DESCRIPTION
**Pull request by araratthehero**
## Description
Now a disclaimer is shown when user clicks "Continue" without making a selection for UPI intent.

## Checklist 
- [x] PR is labelled 
- [x] Code is unit tested
- [x] Changes are tested manually

COAND-946

### CODE REVIEW

File: RELEASE_NOTES.md [hunk 1]
```diff
@@ -9,19 +9,4 @@
...
-  |--------------------------------------------------------------------------------------------------------------|-------------------------------|
-  | [Adyen 3DS2](https://github.com/Adyen/adyen-3ds2-android/releases/tag/2.2.19)                                | **2.2.19**                    |
-  | [Kotlin](https://github.com/JetBrains/kotlin/releases/tag/v1.9.24)                                           | **1.9.24**                    |
-  | [Android Gradle plugin](https://developer.android.com/build/releases/gradle-plugin)                          | **8.4.1**                     |
-  | [AndroidX Compose Compiler](https://developer.android.com/jetpack/androidx/releases/compose-compiler#1.5.14) | **1.5.14**                    |
-  | [Kotlin coroutines](https://github.com/Kotlin/kotlinx.coroutines/releases/tag/1.8.1)                         | **1.8.1**                     |
-  | [AndroidX Fragment](https://developer.android.com/jetpack/androidx/releases/fragment#1.7.1)                  | **1.7.1**                     |
+- For UPI Intent a disclaimer will be shown when "Continue" button is pressed without selecting any UPI option.
```
Comment (jreij) [inline RELEASE_NOTES.md:12]: "disclaimer" seems a bit confusing to me, it's more of a validation error message, what do you think?

Comment (jreij) [inline RELEASE_NOTES.md:12]: Not sure if this should be under "new", feels more like a fix/improvement rather than a new feature (same for the PR label)

Comment (araratthehero) [reply RELEASE_NOTES.md:12]: We do not really have a good label for this. The ones that I am thinking of are `Enhancement` and `Fix`. But `Enhancement` indicates a new feature request and the `Fix` indicates a fix. This is more of an improvement of a previously implemented feature.

image

Comment (jreij) [reply RELEASE_NOTES.md:12]: Right, `Enhancement` is for issues not PRs. I think `Fix` applies here even though it's not exactly a bug fix. Think about `Feature` as anything that includes new functionality (mostly adds new things to the public API but not always) and `Fix` as anything that doesn't directly affect integrations but improves the flow for merchants.

Comment (jreij) [reply RELEASE_NOTES.md:12]: I agree that the naming might be confusing but we mostly use these 2 labels to know if we should make a minor release or a patch, they are not meant to be completely exhaustive to every use case.

**Review by jreij — APPROVED** [review]

**Pull request merged**